edenia/antelope-tools
# Fix/tps and tpb graphic

### Fix TPS and TPB graphic

### What does this PR do?

- Fix bug that queries get_block more than once every 500 milliseconds and includes repeated blocks.
- Add current cpu and net usage, using the data of the last TPS.
- Add Stress Test Dashboard section, including the graphic of transactions.
- Fix query error in check_history_by_endpoint query.

### Steps to test

1. Run the project locally.
2. Go to the home section.
3. Check the graphic and resource utilization.

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/8b9ba8a5-40be-4f5d-beb3-2910a98b2959)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/6fe80050-339c-4f86-ae15-2ae4995988f7)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/bb82aee6-62bb-4411-a91f-a51ab68fff27)
